packages/web-components/@sfx/base/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+/* istanbul ignore file */
+
+export function makeSlot(name?: string) {
+  const slot = document.createElement('slot');
+  if (name) {
+    slot.name = name;
+  }
+  return slot;
+}
+
+export function createChildrenObserver(root: any) {
+  return new MutationObserver((mutations) => {
+    mutations.forEach((mutation) => {
+      let mutedNodes = Array.from(mutation.removedNodes)
+      mutedNodes.forEach((node) => {
+        if (node.nodeType !== Node.COMMENT_NODE) {
+          root.appendChild(node);
+        }
+      });
+    });
+  });
+}
+
+/**
```

Comment:
Put this comment at the top so it's easier to see.

Also, since this isn't a doc comment, it should start with only one `*`.